charts/kafka/README.md

```diff
@@ -1,10 +1,19 @@
 # Apache Kafka Helm Chart
```

Comment:
I expected to see something added to this readme about the metrics, thats why I had Jeff provide this information here:

https://github.com/terascope/helm-charts/issues/1#issuecomment-2722653188

The logic being, this PR takes something away (that other exporter) without explaining what the user is meant to do without that.  If we mean to kick this task down the road, that's fine.  But lets not forget.

Reply:
It's possible the scope of this is larger than just this case, like maybe we need to add a whole section documenting how to use the metrics.  Likely you don't know how to use those things.

Reply:
I agree that we need to add more documentation around metrics. I added an issue: #4 . I don't think I know enough about the exporters to write good docs right now.